hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStreamContext.java

```diff
@@ -115,6 +124,8 @@ public AbfsInputStreamContext build() {
       readAheadBlockSize = readBufferSize;
     }
     // Validation of parameters to be done here.
+    Preconditions.checkArgument(readAheadRange > 0,
```

Comment:
why aren't we allowed a range of 0?

Reply:
It doesn't makes sense right? If set to 0 means disabling readAhead.

Reply:
understood